packages/connect-thegraph/src/core/GraphQLWrapper.ts

```diff
@@ -1,67 +1,63 @@
 import fetch from 'isomorphic-unfetch'
-import ws from 'isomorphic-ws'
 import {
   Client,
-  defaultExchanges,
-  subscriptionExchange,
-  createRequest,
+  GraphQLRequest,
+  createRequest as createRequestUrql,
 } from '@urql/core'
-import { SubscriptionClient } from 'subscriptions-transport-ws'
 import { DocumentNode } from 'graphql'
 import { pipe, subscribe } from 'wonka'
 import { SubscriptionHandler } from '@aragon/connect-types'
-import { ParseFunction, QueryResult, SubscriptionOperation } from '../types'
-
-const AUTO_RECONNECT = true
-const CONNECTION_TIMEOUT = 20 * 1000
-
-function filterSubgraphUrl(url: string): [string, string] {
-  if (!/^(?:https|wss):\/\//.test(url)) {
-    throw new Error('Please provide a valid subgraph URL')
+import { ParseFunction, QueryResult } from '../types'
+
+// Average block time is about 13 seconds on the 2020-08-14
+// See https://etherscan.io/chart/blocktime
+const POLL_INTERVAL = 13 * 1000
+
+function createRequest(query: DocumentNode, args: object): GraphQLRequest {
+  // Make every operation type a query, until GraphQL subscriptions get added again.
+  if (query.definitions) {
+    query = {
+      ...query,
+      definitions: query.definitions.map((definition) => ({
+        ...definition,
+        operation: 'query',
+      })),
+    }
   }
-  return [url.replace(/^wss/, 'https'), url.replace(/^https/, 'wss')]
+  return createRequestUrql(query, args)
 }
 
 export default class GraphQLWrapper {
   #client: Client
   #verbose: boolean
-  close: () => void
 
   constructor(subgraphUrl: string, verbose = false) {
```

Comment:
Maybe having a new argument to provide the `POLL_INTERVAL` that the GraphQLWrapper will use.